psamm/datasource/sbml.py

```diff
@@ -768,6 +774,17 @@ def _add_gene_list(self, parent_tag, gene_id_list):
             gene_tag.set(_tag('id', FBC_V2), id)
             gene_tag.set(_tag('label', FBC_V2), gene_id_list[id])
 
+    def _transform_param(self, stri):
+        if '.' in stri:
+            stri = ''+stri.replace('.', '_')
+        elif '_' in stri:
+            stri = ''+stri.replace('_', '.')
+        if '-' in stri:
+            stri = ''+stri.replace('-', 'neg')
+        elif 'neg' in stri:
+            stri = ''+stri.replace('neg', '-')
+        return stri
```

Comment:
This seems strange. Why do you do both the forward and reverse conversions. Also, why `''+...`?